rusk-recovery/src/bin/keys.rs

```diff
@@ -37,7 +37,7 @@ struct Cli {
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let args = Cli::parse();
     task::run(
-        || rusk_recovery_tools::keys::exec(args.keep),
+        || Ok(rusk_recovery_tools::keys::exec(args.keep)?),
```

Comment:
It's indeed because `task::run` expects the error type `Box<(dyn std::error::Error + 'static)>`, but `exec` returns a simple `io::Error` now.

I can:
1. leave wrapped in `Ok(...)?`
2. change `exec` in recovery to return `Box<(dyn std::error::Error + 'static)>`
3. change `task::run` to return `io::Error`

Reply:
It's ok to leave it wrapped in `Ok(...)?` then. Lets go for 1.